common/models/types/index.d.ts

```diff
@@ -143,17 +143,31 @@ declare interface LexicalModel {
    */
   readonly punctuation?: LexicalModelPunctuation;
 
+  /**
+   * Returns the wordbreaker defined for the model (if it exists).
+   */
+  wordbreaker?: WordBreakingFunction;
+
+  /**
+   * **NOTE:  _Deprecated!_**
```

Comment:
We need to explain what the alternative is and why this one is deprecated 😁 -- it's a little ironic that the deprecated function has more detailed notes than the new one! Also helpful to note the version of Keyman in which it is deprecated (i.e. 14.0).